flow/templates/print_status.jinja

```diff
@@ -0,0 +1,56 @@
+{# The following variables are available to all scripts. #}
+{% if overview %}
+    {% set field_progress_bar = ("%%-%ss  %%-%ss" | format(column_width_label, column_width_bar)) %}
+{% endif %}
+{% if detailed %}
+    {% set column_width_operation = [column_width_operation, 5] | max %}
+    {% set field_job_id = ("%%-%ss  " | format(column_width_id)) %}
+    {% set field_operation = ("%%-%ss %%3s  " | format(column_width_operation)) %}
+    {% set field_operation_title = ("%%-%ss  " | format(column_width_operation+4)) %}
+    {% set field_label = ("%%-%ss" | format(column_width_total_label)) %}
+    {% set ns = namespace(dash='', field_parameters='') %}
+    {% if parameters %}
+        {% for para in parameters %}
+            {% set ns.dash = ns.dash + ("%s  " | format('-' * column_width_parameters[loop.index0])) %}
+            {% set ns.field_parameters = ns.field_parameters + ("%%-%ss  " | format(column_width_parameters[loop.index0])) %}
+        {% endfor %}
+        {% set para_head = ns.field_parameters | format(*parameters) %}
+    {% endif %}
+    {% set field_head = field_job_id + field_operation_title + '%s' + field_label %}
+{% endif %}
+
+{% block overview %}
+{% if overview %}
+# Overview:
+Total # of jobs: {{ '%s' | format(jobs | length) }}
+
+{% block progress %}
+{{ field_progress_bar | format('label', 'ratio') }}
+{{ field_progress_bar | format('-' * column_width_label, '-' * column_width_bar) }}
+{% for label in progress_sorted %}
+{{ field_progress_bar | format(label[0], label[1] | draw_progressbar(jobs | length)) }}
+{% endfor %}
+{% endblock%}
+{% endif %}
+{% endblock%}
+
+{% block detailed %}
+{% if detailed %}
+# Detailed view:
+{{ field_head | format('job_id', 'operation', para_head, 'labels') }}
+{{ field_head | format('-' * column_width_id, '-' * (column_width_operation+4), ns.dash, '-' * column_width_total_label) }}
+    {% for job in jobs %}
+        {% if parameters %}
+            {% set para_output = ns.field_parameters | format(*job['parameters'].values()) %}
+        {% endif %}
+        {% for key, value in job['operations'].items() if value | job_filter(scheduler_status_code, all_ops) %}
+            {% if loop.first %}
+{{ field_job_id | format(job['job_id']) }}{{ field_operation | highlight(value['eligible']) | format(key, '['+scheduler_status_code[value['scheduler_status']]+']') }}{{para_output}}{{ '%s' | format(job['labels'] | join(', ')) }}
+            {% else %}
+{{ field_job_id | format('')}}{{ field_operation | highlight(value['eligible']) | format(key, '['+scheduler_status_code[value['scheduler_status']]+']') }}{{para_output}}{{ '%s' | format(job['labels'] | join(', ')) }}
+            {% endif %}
+        {% endfor %}
+    {% endfor %}
+{{ status_legend }}
```

Comment:
I currently tried adding indent in only this template so we could see if it looks good and I can change all once we finalized a proper format. Currently only logic codes are indented({% %}), (output lines ({{ }}) are not, I had some issue make them work properly)

Reply:
@bdice Would like your input on what's the best way to properly format the template code.